python/src/scipp/compat/mantid.py

```diff
@@ -423,15 +423,14 @@ def _convert_MatrixWorkspace_info(ws):
             make_run(ws),
             "sample":
             make_sample(ws),
-            "rotation":
-            rot,
-            "shape":
-            shp,
             "instrument-name":
             sc.Variable(
                 value=ws.componentInfo().name(ws.componentInfo().root()))
         },
     }
+    if 'spectrum' in spec_coord.dims:
+        info["attrs"].update({"rotation": rot, "shape": shp})
```

Comment:
Can't we just use the correct dimension in `rot` and `shp` instead, or is there anything conceptually preventing this?

Reply:
In the case of the failing test, the spectrum do not correlate to anything geometrically varying. The `spectrum` check should be robust, however it is easily possible for is to be scattering angle, etc, so I'll introduce what should be a better check.